AppCenterDistribute/AppCenterDistribute/Internals/Ingestion/MSDistributeIngestion.m

```diff
@@ -59,7 +59,9 @@ - (void)willSendHTTPRequestToURL:(NSURL *)url withHeaders:(nullable NSDictionary
     }];
 
     // Log URL and headers.
-    MSLogVerbose([MSAppCenter logTag], @"URL: %@", url);
+    NSString *encodeUrl = [MSHttpUtil hideSecretInString:url.absoluteString
+                                                  secret:self.appSecret];
```

Comment:
This should be inlined in `MSLogVerbose` to avoid computing it when verbose logs are disabled